src/ResourceManager/DataFactories/Commands.DataFactoryV2/IntegrationRuntimes/GetAzureDataFactoryIntegrationRuntimeNodeIpCommand.cs

```diff
@@ -0,0 +1,46 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using System.Security.Permissions;
+using Microsoft.Azure.Management.DataFactory.Models;
+
+namespace Microsoft.Azure.Commands.DataFactoryV2
+{
+    [Cmdlet(
+        VerbsCommon.Get,
+        Constants.IntegrationRuntimeNodeIp,
+        DefaultParameterSetName = ParameterSetNames.ByIntegrationRuntimeName),
+        OutputType(typeof(IntegrationRuntimeNodeIpAddress))]
+    public class GetAzureDataFactoryIntegrationRuntimeNodeIpCommand : IntegrationRuntimeContextBaseCmdlet
```

Comment:
See my response in the review:  https://github.com/Azure/azure-powershell-pr/wiki/ADF-V2-Integration-Runtime-New-Cmdlets-(December-2017)-Notes#response 

If this is an expensive call, then you should provide an option switch on the existing Get-* cdlet to include this ifnormation, ratehr than havign a new cmdlet.

Reply:
Changed it as your comments. Thanks.